docs/appendices/release-notes/unreleased.rst

```diff
@@ -113,3 +113,6 @@ Fixes
   ``CREATE TABLE`` statements to be rejected because the validation always used
   a fixed value of ``5`` instead of the actual number of shards declared within
   the ``CREATE TABLE`` statement.
+
+- Fixed an issue that threw an exception when ``ORDER BY`` clauses contain
+  aliased literals.
```

Comment:
```suggestion
  the output column value of an aliased column.
```

Reply:
Please check if this suggestion may explain the case in a better way.

Reply:
Thanks, I made one more change on top of your suggestion, `the output column position or the alias name of an aliased column`
because it also fixes 
```sql
select 123 as alias from t order by alias
```